src/schema/clan/remove.ts

```diff
@@ -0,0 +1,21 @@
+import { Type } from "@sinclair/typebox";
+
+import { defineEndpoint } from "@/generator-helpers.js";
+
+export default defineEndpoint({
+    source: "user",
+    target: "server",
+    description: "Remove your clan.",
+    request: {
+        data: Type.Object({
+            userId: Type.Ref("userId"),
+            clanId: Type.Ref("clanId"),
+        }),
+    },
+    response: [
+        { status: "success" },
+        { status: "failed", reason: "no_member" },
+        { status: "failed", reason: "permission_denied" },
+        { status: "failed", reason: "clan_not_found" },
```

Comment:
`no_member` and `clan_not_found` are invalid reasons. There is always at least one member in a clan.